### ISSUE TITLE
Install and explain a PR bot

### ISSUE THREAD

**Issue by olleolleolle** [RailsGirlsCPH/mentor-mentee-platform#21]
This Issue is about learning to use GitHub-related tools to **manage software dependencies** in projects.

**Problem**: Software often has bugs, and then new versions of packages are released, with a warning "don't use the old version!" and we as developers have to update Gemfile.lock or package-lock.json or what have you. This can be a lot of work, and it can be a little repetitive. And, sometimes the bug is an important bug, a vulnerability affecting security.

**Solutions**:

- Always be vigilant (difficult, because we're humans, we miss things, we make mistakes)
- Automation (tricky to build – _but_, some people who already built it share it cost-free with Open Source projects – and this project is Open Source)

Examples of an offering of Ruby package manager auto-updater: [Dependabot](https://dependabot.com/#pricing). 

- [x] Sign up for dependabot
- [ ] Configure the directory with a Rails app to be a watched directory of Ruby files (so Dependabot can find it)
- [ ] Add a section to the README about Automations added to the GitHub repo

**Comment by nynnejc** [comment]
Really like the depebdabot solution. Thanks so much for sketching up the issues leading to this, @olleolleolle

**Comment by sibsmc** [comment]
Requested Dependabot, awaiting approval.

**Comment by nynnejc** [comment]
I just approved! Thanks for installing

**Comment by sibsmc** [comment]
Dependabot now creates pull requests to update packages. I will update the readme in Server. 
I presume we need some kind of test to run to make sure that our app will still run with these updates. So should we put some kind of testing in place before we start merging these updates?  
@zoeyzou we can add the client directory to depandabot. I think it then reviews package.json and creates a pull request for new versions of Javascript packages which are available. I wanted to check first if you think that sounds like a good idea.

**Comment by diwanow** [comment]
Good idea for the validation - it should be covered in #42 Meanwhile, I am thinking we can validate the PRs manually - if we have a setup, I guess just starting the app or even just building it should be enough and there are not that many PRs. 

Maybe we can settle on picking them one by one by assigning to yourself, checking out the branch, building locally and if ok, approval and merge. How does that sound?

**Comment by diwanow** [comment]
hey @sibsmc is there anything left to complete here on backend side? I think the task with the README from the description of the issue might be the only thing missing.

I created a separate issue for using dependabot for front-end code - the team can consider its value there.

**Comment by sibsmc** [comment]
Closed as dependabot now installed and a small explanation added to readme file.
